.github/workflows/linters.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - run: sudo npm i -g npm@7
```

Comment:
why `sudo`?

Reply:
because it will install global package to

```
/usr/local/bin/npx -> /usr/local/lib/node_modules/npm/bin/npx-cli.js
/usr/local/bin/npm -> /usr/local/lib/node_modules/npm/bin/npm-cli.js
```